### DISCUSSION TITLE
How to stop showing collaborators activity?

### DISCUSSION THREAD

**Discussion by Yura52** [community/community#30863] [Other Feature Feedback, Questions, & Ideas]
Hi! In my feed, there are numerous posts like "User X contributed to repository Y" with the explanation being "You're seeing this because you collaborated with User X". This effectively breaks my feed, because I am not interested in this activity. How can I turn it off?

P.S. What is even worse is that in my case, User X is a complete stranger to me. So my feed is flooded from time to time with the activity of a random user who I don't even know.

**Comment by jfullstackdev** [comment]
as of now, I think the workaround is to block him

you may want to check whether he/she will still appear on the feed after 24 hours,

**Reply by KeterSCP** [reply]
Are there any plans to add a specific feature/filter for this, so we don't have to **block** users just to hide their activity from the feed?

**Reply by jfullstackdev** [reply]
yup, if I remember that correctly, like there will be a button for that, `I don't want to see these kinds of posts`, the idea is like that

**Reply by PixlRainbow** [reply]
I work on 20+ different repositories that can each have as many as 100 collaborators, so my feeds are just flooded with strangers, entirely flushing out activity from people I follow and repositories I star.
Under the new "show less activity like this", **there is no option to mute activity from ALL collaborators** -- only to mute from particular users, repos or actions. 
![image](https://user-images.githubusercontent.com/44422178/198864323-6648f99d-d8c5-4a20-8717-7302a7b36120.png)

**If I have to individually mute each collaborator, I would have to look through my feed and mute hundreds of people one by one.**

**Comment by luni3359** [comment]
This update is awful, I don't care about these collaborator recommendations either please do something about it.
